backend/src/v5/schemas/tickets/index.js

```diff
@@ -54,13 +54,13 @@ const generatePropertiesValidator = async (teamspace, project, model, templateId
 			if (prop.values) {
 				let values;
 				switch (prop.values) {
-				case presetEnumValues.JOBS_AND_USERS:
+				case presetEnumValues.ROLES_AND_USERS:
 					{
 						const excludeViewers = prop.name === basePropertyLabels.ASSIGNEES;
 						const users = await getUsersWithPermissions(teamspace, project, model, excludeViewers);
-						const accessibleJobs = await getJobsByUsers(teamspace, users);
+						const accessibleRoles = await getRolesByUsers(teamspace, users);
 
-						values = [...accessibleJobs, ...users];
+						values = [...accessibleRoles.map(UUIDToString), ...users];
```

Comment:
are the role IDs converted to UUID at any point? we want make sure we store UUIDs on the tickets. I don't thnk I see a conversion

There's also need to be a convertion on the middleware on the response in GET requests for tickets